src/elements/Button/ButtonGroup.js

```diff
@@ -17,21 +17,33 @@ import {
  */
 function ButtonGroup(props) {
   const {
-    attached, basic, children, className, color, icon, labeled, size, vertical, widths,
+    attached, basic, children, className, color, compact, disabled,
+    fluid, icon, inverted, labeled, negative, positive, primary,
+    secondary, size, toggle, vertical, widths,
   } = props
 
   const classes = cx('ui',
     size,
     color,
     useValueAndKey(attached, 'attached'),
     useKeyOnly(basic, 'basic'),
+    useKeyOnly(compact, 'compact'),
+    useKeyOnly(disabled, 'disabled'),
```

Comment:
Looks like they're all good, except for `disabled`.  I can find [no css for disabled `.ui.buttons`](https://github.com/Semantic-Org/Semantic-UI/blob/master/src/definitions/elements/button.less#L196) and the markup does not seem to work either:

![image](https://cloud.githubusercontent.com/assets/5067638/19445846/dc5e54c2-944a-11e6-90ee-856c9ba3f358.png)

I'll pull this one and merge.